roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
@detiber, we're open to ideas.  Our preference is for options that don't result in users having to answer questions or supply arguments when.  Somehow we want to guess what they are likely to want to install simply by running the tool.  Obviously, there are less common edge-cases and for those they can use settings.  Right now we're thinking a single pre-flight container image can exist.